lib/screens/more_details_page.dart

```diff
@@ -289,44 +113,100 @@ class _MoreDetailsState extends State<MoreDetails> {
         appBar: AppBar(),
       ),
       body: _moreDetailModel.question == null ||
-              _moreDetailModel.user == null ||
-              _moreDetailModel.userAnswer == null
+              _moreDetailModel.question == null ||
+              (_moreDetailModel.userAnswer.length == 0
+                  ? false
+                  : _moreDetailModel.userAnswer[0].user == null) ||
+              (_moreDetailModel.userAnswer.length == 0
+                  ? false
+                  : _moreDetailModel.userAnswer[0].answer == null)
           ? Container(
               child: Center(
                 child: CircularProgressIndicator(),
               ),
             )
-          : ListView(
-              children: [
-                Container(
-                  margin: EdgeInsets.fromLTRB(10, 10, 10, 0),
-                  padding: EdgeInsets.fromLTRB(3, 0, 3, 0),
-                  decoration: BoxDecoration(
-                    color: Colors.grey[300],
-                    borderRadius: BorderRadius.all(
-                      Radius.circular(20.0),
-                    ),
+          : SingleChildScrollView(
+              child: Container(
+                constraints: BoxConstraints(
+                  minHeight: MediaQuery.of(context).size.height - 269.0,
+                ),
+                margin: EdgeInsets.fromLTRB(10, 10, 10, 0),
+                padding: EdgeInsets.fromLTRB(3, 0, 3, 0),
+                decoration: BoxDecoration(
+                  color: Colors.grey[300],
+                  borderRadius: BorderRadius.all(
+                    Radius.circular(20.0),
                   ),
+                ),
+                child: Padding(
+                  padding: const EdgeInsets.fromLTRB(20, 0, 20, 10),
                   child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
                     children: [
-                      _headerStack(),
-                      _questionFieldViewer(),
+                      MoreDetailsView(
+                        moreDetailModel: _moreDetailModel,
+                        isAnswer: false,
+                        isCurrUser: _moreDetailModel.user.id == currUserId,
+                      ),
+                      Padding(
+                        padding:
+                            const EdgeInsets.fromLTRB(0.0, 15.0, 0.0, 15.0),
+                        child: Divider(
+                          color: Colors.black,
+                          thickness: 2.0,
+                          height: 0.0,
+                        ),
+                      ),
+                      Text(
+                        "Answers",
```

Comment:
Consider adding a counter to the `Answers` instead of hardcoding it like this. So each time a new answer is posted under the question, it will show **3 Answers** etc.